### PR TITLE
[zh] sync concepts/services-networking/_index endpoint-slice-v1

### PR DESCRIPTION
**Pull request by asa3311**

### CODE REVIEW

File: content/zh-cn/docs/reference/kubernetes-api/service-resources/endpoint-slice-v1.md [hunk 1]
```diff
@@ -300,6 +302,8 @@ EndpointSlice 是实现某 Service 的端点的子集。一个 Service 可以有
 
     * Kubernetes 定义的前缀名称：
       * 'kubernetes.io/h2c' - 使用明文的 HTTP/2 协议，详见 https://www.rfc-editor.org/rfc/rfc7540
+      * 'kubernetes.io/ws' - WebSocket 通过明文文传输，详见 https://www.rfc-editor.org/rfc/rfc6455
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/service-resources/endpoint-slice-v1.md:305]:
```suggestion
      * 'kubernetes.io/ws' - 通过明文传输的 WebSocket，详见 https://www.rfc-editor.org/rfc/rfc6455
```

File: content/zh-cn/docs/reference/kubernetes-api/service-resources/endpoint-slice-v1.md [hunk 2]
```diff
@@ -300,6 +302,8 @@ EndpointSlice 是实现某 Service 的端点的子集。一个 Service 可以有
...
+      * 'kubernetes.io/wss' - WebSocket 通过 TLS 传输，详见 https://www.rfc-editor.org/rfc/rfc6455
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/service-resources/endpoint-slice-v1.md:306]:
```suggestion
      * 'kubernetes.io/wss' - 通过 TLS 传输的 WebSocket，详见 https://www.rfc-editor.org/rfc/rfc6455
```

File: content/zh-cn/docs/concepts/services-networking/_index.md [hunk 3]
```diff
@@ -106,6 +108,8 @@ Kubernetes 网络解决四方面的问题：
   以支持来自于集群外部的访问。
   - [Ingress](/zh-cn/docs/concepts/services-networking/ingress/)
     提供专门用于暴露 HTTP 应用程序、网站和 API 的额外功能。
+  - [Gateway API](/zh-cn/docs/concepts/services-networking/gateway/) 
+    是一个提供富有表现力、可扩展并以角色为导向的 API 种类的 {{<glossary_tooltip text="插件" term_id="addons">}}。
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/services-networking/_index.md:112]:
```suggestion
    是一个{{<glossary_tooltip text="插件" term_id="addons">}}，
    为服务网络建模提供富有表现力、可扩展和面向角色的 API 系列类别。
```

**Review by ydFu — APPROVED** [review]
/approve

**Pull request merged**